uint32 width
uint32 height
bool saveInterface
bool show_nav_obstacles
bool show_nav_distance_grid
bool show_nav_visibility_grid
bool show_nav_hidzones_grid
bool floor
bool walls
bool tiles
bool shadows

